docs/bsp.md

```diff
@@ -864,6 +864,9 @@ trait ScalaBuildTarget {
 
   /** The target platform for this target */
   def platform: Int
+  
+  /** A sequence of Scala jars. */
```

Comment:
Can you elaborate on what jars these are? scala-library, scala-reflect, scala-compiler?

Reply:
the `scalac` command line interface does not have a --scala-jars flag so it's not obvious to me  what this is